### PR TITLE
Add ci

### PR DESCRIPTION
**Pull request by MauroToscano**
Add CI with build and lint

### CODE REVIEW

File: .github/workflows/build.yml [hunk 1]
```diff
@@ -0,0 +1,35 @@
...
+        with:
+          version: v1.54
+      - name: Checkout code
+        uses: actions/checkout@v2
+      - name: Build aggregator
+        run: go build aggregator/cmd/main.go
+      - name: Build operator
+        run: go build operator/cmd/main.go
+      - name: Build task sender
+        run: go build operator/cmd/main.go
```
Comment (entropidelic) [inline .github/workflows/build.yml]:
```suggestion
        run: go build task_sender/cmd/main.go
```

File: .github/workflows/golangci-lint.yml [hunk 2]
```diff
@@ -0,0 +1,55 @@
...
+          # skip-cache: true
+
+          # Optional: if set to true, then the action won't cache or restore ~/go/pkg.
+          # skip-pkg-cache: true
+
+          # Optional: if set to true, then the action won't cache or restore ~/.cache/go-build.
+          # skip-build-cache: true
+
+          # Optional: The mode to install golangci-lint. It can be 'binary' or 'goinstall'.
+          # install-mode: "goinstall"
```
Comment (entropidelic) [inline .github/workflows/golangci-lint.yml:55]: are these comments intended to be left just in case?

**Review by entropidelic — APPROVED** [review]

File: .github/workflows/build.yml [hunk 3]
```diff
@@ -0,0 +1,27 @@
...
+  build:
+    runs-on: ubuntu-latest
+    
+    steps:
+      - uses: actions/checkout@v4
+      - uses: actions/setup-go@v5
+        with:
+          go-version: '1.22'
+          cache: false
+      - name: Checkout code
```
Comment (NicolasRampoldi) [inline .github/workflows/build.yml:20]: The first "uses" step checksout, do we need to do it again?

**Review by NicolasRampoldi — APPROVED** [review]

File: .github/workflows/build.yml (hunk 3 above)
Comment (MauroToscano) [reply .github/workflows/build.yml:20]: Probably not

**Pull request merged**